### DISCUSSION TITLE
What font best emulates default Windows 11 taskbar clock?

### DISCUSSION THREAD

**Discussion by e-t-l** [martinet101/ElevenClock#1396] [Q&A]
![image](https://github.com/user-attachments/assets/20c4504c-2be8-4d22-978f-6e9d8143410f)

This is my default Windows 11 taskbar clock. Notice it has the crossbar at the bottom of the number "1", which Segoe UI and Segoe UI Variable fonts do not have. I've read that this is an older version of the Segoe UI Variable font; in that case, is there a way to download that older version, or is there an identical font that could be used instead?

**Comment by marticliment** [comment]
I am aware of this, but I have never been able to find which exact font the Windows 11 Clock runs on.

**Comment by wrbird** [comment]
Try Calibri
<img width="105" height="67" alt="image" src="https://github.com/user-attachments/assets/c2d169c9-6ffd-4bd0-b9f5-02b1b488b540" />

**Reply by marticliment** [reply]
Calibri is not. It is a special variation of Segoe UI
